ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/State.hs

```diff
@@ -150,21 +152,23 @@ reconstructSummaryLedger cfg@HardForkLedgerConfig{..} st =
 -- NOTE: The resulting 'EpochInfo' is a snapshot only, with a limited range.
 -- It should not be stored.
 epochInfoLedger :: All SingleEraBlock xs
-                => HardForkLedgerConfig xs
+                => RelativeTime
+                -> HardForkLedgerConfig xs
                 -> HardForkState LedgerState xs
                 -> EpochInfo Identity
-epochInfoLedger cfg st =
-    History.snapshotEpochInfo $
+epochInfoLedger sysRelTime  cfg st =
```

Comment:
However, if I try to change the compile errors that result from adding a `RelativeTime` argument here, I end up fairly deep in the call chain.